js/main.js

```diff
@@ -0,0 +1,26 @@
+$(function () {
+
+  $("#stopButton").click(illuminateRed);
+  function illuminateRed() {
+    clearLights();
+    $("#stopLight").css("backgroundColor", "red");
+  }
+
+  $("#slowButton").click(illuminateYellow);
+  function illuminateYellow() {
+    clearLights();
+    $("#slowLight").css("backgroundColor", "yellow");
+  }
+
+  $("#goButton").click(illuminateGreen);
+    function illuminateGreen() {
+      clearLights();
+      $("#goLight").css("backgroundColor", "green");
+    }
+
+  function clearLights() {
+    $("#stopLight").css("backgroundColor", "black");
```

Comment:
@eddiejew the selector could have been `$('.bulb').css("backgroundColor", "black");` and this would have applied to all three lights which have the bulb class.